RHEL/7/input/oval/service_nails_enabled.xml

```diff
@@ -0,0 +1,99 @@
+<def-group>
+ <!-- THIS FILE IS GENERATED by create_services_enabled.py.  DO NOT EDIT.  -->
+  <definition class="compliance" id="service_nails_enabled"
+  version="1">
+    <metadata>
+      <title>Service nails Enabled</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 6</platform>
```

Comment:
This looks good, just the platform should be RHEL-7, otherwise this OVAL won't be counted / included. I will fix it by manual merge. Thanks for the change!